search/search_repository_whitebox_test.go

```diff
@@ -144,90 +141,82 @@ func (s *searchRepositoryWhiteboxTest) TestSearchByText() {
 			minimumResults: 1,
 		},
 	}
+	// create the parent space
+	fxt := tf.NewTestFixture(s.T(), s.DB, tf.Spaces(1))
+	//
+	for _, testData := range testDataSet {
+		workItem := testData.wi
+		searchString := testData.searchString
+		minimumResults := testData.minimumResults
+		workItemURLInSearchString := "http://demo.almighty.io/work-item/list/detail/"
+		req := &http.Request{Host: "localhost"}
+		params := url.Values{}
+		ctx := goa.NewContext(context.Background(), nil, req, params)
+		createdWorkItem, err := wir.Create(ctx, fxt.Spaces[0].ID, workitem.SystemBug, workItem.Fields, s.modifierID)
```

Comment:
Sorry but I have to reject this change because you're mixing up a newly defined test fixture space with a WIT that belongs to the system space. That only works because of a hack. Can you instead maybe just also ask for a WIT in your test fixture above (`tf.WorkItemTypes(1)`)? Then you can use that WIT's ID instead of `workitem.SystemBug`. Right now the two WITs (the new one and `workitem.SystemBug`) are compatible.

Reply:
ok, let me check that

Reply:
fixed: I'm now using a WorkItemType created by the TestFixture.